### DISCUSSION TITLE
Aesthetic changes

### DISCUSSION THREAD

**Discussion by Ark-SFL** [sunflower-land/sunflower-land#4222] [Ideas]
### I think we could adjust some aesthetic aspects within the game, here are some small suggestions. I hope you evaluate.

**Change the XP symbol of buildings:**
arrow up ↑ for ![image](https://github.com/user-attachments/assets/5c587450-cb2c-4465-ae57-dc236a452b99)

![image](https://github.com/user-attachments/assets/a52528de-87d0-4d8c-b224-4020d93b66d5) -> ![image](https://github.com/user-attachments/assets/3c271008-3b03-4bfb-a429-ca7172fc1d6d)

**Change the symbol of fertilizers when they are used on crops/fruits:**
_up arrow for each specific representation_
Sprout Mix -> ![image](https://github.com/user-attachments/assets/8d67a387-73fd-4990-a2c5-aa47e0c5dc84)
![image](https://github.com/user-attachments/assets/815cb794-06da-4cc6-905e-14cbd93e098d) -> ![image](https://github.com/user-attachments/assets/f664a127-ae91-40b7-8279-8d95ed52cbdd)

**When Time Warp Totem or any Hourglass are activated on the island, would it be possible to add some visual time counter information?**
![image](https://github.com/user-attachments/assets/ac1fc3dc-4d15-4c51-8d9c-3bcd0bb96b82)
![image](https://github.com/user-attachments/assets/057b11d9-34db-45b3-b1aa-583bd60b9850)

**Players with 4 wells built can create a single well. (The well as an example is the one at Goblin Retreat, from the liquid pool.)**
![image](https://github.com/user-attachments/assets/2ce51804-70d7-47b9-86fa-823db4dbd6d7) -> ![image](https://github.com/user-attachments/assets/8cc83fdd-e2d9-4a12-b045-0e82bcc821dc)

**Comment by Polysemouse** [comment]
I agree the up arrow isn't the best icon for XP, however XP doesn't really feel the right icon when playing around it in code (because it says [XP ICON] 32.01 XP).  Something else is needed but I am not sure what is the best icon.

**Reply by Ark-SFL** [reply]
Hello Polymouse, I hope you are well!

In terms of code, I really don't know how it works or how it could help, I just think they could change the up arrow.
The XP image I took from the pet's streak tab.
